### DISCUSSION TITLE
Website source

### DISCUSSION THREAD

**Discussion by JohnVeness** [memtest86plus/memtest86plus#158] [Q&A] [answered]
Hi. I spotted a couple of typos on memtest.org. Is the website source code in a repo somewhere, so I could submit a PR?

**Comment by x86fr** [comment, marked as answer]
No yet, but you can post them here. I'll correct that.

**Reply by JohnVeness** [reply]
Thanks. On https://memtest.org/archives, the middle section says "These version" which should be plural.

The other two sections say "These binaries" which is grammatically correct, but I wonder if for consistency (and because the third section does include sources), all three sections should say "These versions". Indeed, maybe the warning should just be once, at the top of the page, instead of at the beginning of each section.

Thanks again!

**Reply by JohnVeness** [reply]
On https://memtest.org/

- "an unified" should be "a unified"
- "make sure there is no memory issues" should be "make sure there are no memory issues"
- "Which medias are supported?" should be "Which media are supported?" (media is already plural)
- In several places, "an USB" should be "a USB"
- In "The Memtest86+ binary is actually not signed by Microsoft", I would remove the word "actually" as unnecessary
- In "To boot on a USB Key" I would say "boot from". Also, you might to check consistency: sometimes you say "USB Flash drive" etc.
- In "You can wait more for even greater confidence, or just remove your USB Drive and press ESC to stop", minor quibble but I think "wait longer" is better. Also, you may have booted from an optical disc or even a floppy. Do you actually need to remove the media anyway? Maybe just say "press ESC to stop" without the removing bit
- "Ie: 16GB of RAM" should be "E.g.: 16GB of RAM"
- In "Please consider the actual code as experimental", again I would remove "actual"
- "Please fill a bug report" should be "Please file a bug report" (or "Please fill in a bug report")

**Reply by JohnVeness** [reply]
On https://memtest.org/dev:
- "Please fill a bug report" should be "Please file a bug report" (or "Please fill in a bug report"), as above
- In "Build is only tested", I think "Building" would be better, or "The build process"

**Reply by JohnVeness** [reply]
I'll leave https://memtest.org/readme for now - and anyway I see that *is* in the repo so I may do a PR there!

**Reply by x86fr** [reply]
Everything should now be corrected, except the second point in https://memtest.org/dev.

Almost all the page at https://memtest.org/dev is dynamically extracted from https://github.com/memtest86plus/memtest86plus/blob/main/README.md and sync'd every night.

**Reply by JohnVeness** [reply]
Many thanks @x86fr .

One thing that got missed:
- On https://memtest.org/, there's one remaining "an USB" in the "How to create a bootable Memtest86+ USB Drive using MacOS" expandable section

Thanks for the info about the README syncing.
